test/controllers/ReviewPenaltyController.test.ts

```diff
@@ -1,13 +1,16 @@
 import 'reflect-metadata';
 
+import { Arg } from '@fluffy-spoon/substitute';
 import { Penalty, PenaltyList } from 'ch-sdk-node/dist/services/lfp';
 import { expect } from 'chai';
-import { INTERNAL_SERVER_ERROR, OK } from 'http-status-codes';
+import { INTERNAL_SERVER_ERROR, OK, UNAUTHORIZED } from 'http-status-codes';
 import request from 'supertest';
+import { createSubstituteOf } from '../SubstituteFactory';
```

Comment:
`app/test/SubstituteFactory`

Reply:
doesnt like it when I include the app, would this be okay for you?
`import { createSubstituteOf } from 'test/SubstituteFactory';`

Reply:
Oh yeah sorry it is 'test/'